docs/api-reference/web/plugins/solana-wallet.mdx

```diff
@@ -103,20 +98,19 @@ After initialising the `torusPlugin`, use the `addPlugin()` function to your Web
 await web3AuthInstance.addPlugin(torusPlugin);
 ```
 
-#### Example
+### Example
 
 ```javascript
-const web3AuthInstance = new Web3Auth({
-  chainConfig: currentChainConfig,
-  clientId, // get your client id from https://dashboard.web3auth.io,
-});
+import { SolanaWalletConnectorPlugin } from "@web3auth/solana-wallet-connector-plugin";
+
 const torusPlugin = new SolanaWalletConnectorPlugin({
   torusWalletOpts: {},
   walletInitOptions: {
+    useWalletConnect: true,
```

Comment:
this option is not supported in solana connector plugin

Reply:
got it

Reply:
removed